ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -128,6 +128,9 @@
 /* Format string for badge label button accessibility label. Format: "<Item Accessibility>, <Badge Label Accessibility>". Example: "Notifications, 5 new notifications" */
 "Accessibility.BadgeLabelButton.LabelFormat" = "%@, %@";
 
+/* Accessibility label for the small x button of the textfield to clear text */
+"Accessibility.TextField.Clear" = "Clear text";
```

Comment:
Is "clear text" standard terminology? If not, I might clarify in the comment something more like "Accessibility label for the small x button of the text field to remove all text from the text field", so it's more clear to the localizers.

Reply:
I can update the comment, but the text is from the UITextField's clear button.